test/IoTSettings.test.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as path from "path";
+import { ConfigKey, OSPlatform } from "../src/constants";
+
+const vscode = require("../__mocks__/vscode");
+const { getHomeDir, getPlatform } = require("../src/utils");
+
+jest.mock("../src/utils");
+jest.mock("../src/configHandler");
+
+let IoTWorkbenchSettingsModule: typeof import("../src/IoTSettings");
```

Comment:
Since we isolate the import of modules, we require the module before each test. We must indicate the type of module before require the module to avoid type error.